releases/release-8.1.2.md

```diff
@@ -0,0 +1,216 @@
+---
+title: TiDB 8.1.2 Release Notes
+summary: 了解 TiDB 8.1.2 版本的兼容性变更、改进提升，以及错误修复。
+---
+
+# TiDB 8.1.2 Release Notes
+
+发版日期：2024 年 x 月 x 日
+
+TiDB 版本：8.1.2
+
+试用链接：[快速体验](https://docs.pingcap.com/zh/tidb/v8.1/quick-start-with-tidb) | [生产部署](https://docs.pingcap.com/zh/tidb/v8.1/production-deployment-using-tiup) | [下载离线包](https://cn.pingcap.com/product-community/?version=v8.1.2#version-list)
+
+## 兼容性变更
+
+- note [#issue](https://github.com/pingcap/${repo-name}/issues/${issue-id}) @[贡献者 GitHub ID](https://github.com/${github-id})
+
+## 改进提升
+
++ TiDB
+
+    - note [#issue](https://github.com/pingcap/tidb/issues/${issue-id}) @[贡献者 GitHub ID](https://github.com/${github-id})
+    - note [#issue](https://github.com/pingcap/tidb/issues/${issue-id}) @[贡献者 GitHub ID](https://github.com/${github-id})
+    - (dup): release-8.3.0.md > # 可观测性 * 添加 Request Unit (RU) 配置监控指标 [#8444](https://github.com/tikv/pd/issues/8444) @[nolouch](https://github.com/nolouch)
+
++ TiKV <!--tw@qiancai: 2 notes-->
+
+    - note [#issue](https://github.com/tikv/tikv/issues/${issue-id}) @[贡献者 GitHub ID](https://github.com/${github-id})
+    - note [#issue](https://github.com/tikv/tikv/issues/${issue-id}) @[贡献者 GitHub ID](https://github.com/${github-id})
+    - (dup): release-8.4.0.md > 改进提升> TiKV - 优化空表和小 Region 场景下 Region Merge 的速度 [#17376](https://github.com/tikv/tikv/issues/17376) @[LykxSassinator](https://github.com/LykxSassinator)
+    - (dup): release-7.5.4.md > 改进提升> TiKV - 优化 TiKV 的 `DiskFull` 检测使之与 RaftEngine 的配置项 `spill-dir` 兼容，确保该特性能够稳定运行 [#17356](https://github.com/tikv/tikv/issues/17356) @[LykxSassinator](https://github.com/LykxSassinator)
+    - (dup): release-7.5.4.md > 改进提升> TiKV - 优化存在大量 DELETE 版本时 RocksDB 的 compaction 触发机制，以加快磁盘空间回收 [#17269](https://github.com/tikv/tikv/issues/17269) @[AndreMouche](https://github.com/AndreMouche)
+    - 新增对配置项 `import.num-threads` 的在线修改支持 [#17807](https://github.com/tikv/tikv/issues/17807) @[RidRisR](https://github.com/RidRisR)
+    - 更新备份恢复侧的 aws 相关 sdk(rusoto) 到官方 sdk. [#12371](https://github.com/tikv/tikv/issues/12371)@[akoshchiy](https://github.com/akoshchiy)
```

Comment:
```suggestion
    - 支持在线修改 `import.num-threads` 配置项 [#17807](https://github.com/tikv/tikv/issues/17807) @[RidRisR](https://github.com/RidRisR)
    - 将备份恢复侧用于访问 Amazon S3 等外部存储的 SDK 更新为 <a href="https://aws.amazon.com/cn/sdk-for-rust/">AWS Rust SDK</a>，替换掉原有的 Rusoto 库，以更好地兼容 AWS 的 IMDSv2 以及 EKS Pod Identity 等新特性 [#12371](https://github.com/tikv/tikv/issues/12371)@[akoshchiy](https://github.com/akoshchiy)
```

Reply:
@3pointer PTAL. Thanks. Note that #12371 seems relevant to the following release note in v8.5.0, so I've used a similar description here. 

![image](https://github.com/user-attachments/assets/9e00ebe0-2d37-4e79-a6ae-c09c54aaeb3d)

Reply:
@BornChanger PTAL. Thanks.